### DISCUSSION TITLE
Error pada pengaplikasian lubridate

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#538] [Q&A] [answered]
Selamat Malam Kaka Instruktur
berikut kendalanya 
![image](https://user-images.githubusercontent.com/123348110/217841483-55ed82da-5b9e-4ad1-b330-d7b72d93c3a6.png)

setiap kali running di awal untuk pertama kali bisa berjalan normal
tetapi ketika di run ke3 sampai seterusnya berubah menjadi NA,
padahal saya mengikuti formatnya sesuai dengan month,date,year, hour,minutes nya.. apa ada solusi ?
terima kasih dan sehat selalu

**Comment by kevbow** [comment, marked as answer]
Dear @DeanAlexander27,

Ketika Anda melakukan pengubahan pertama menggunakan `mdy_hm()`, format dari data telah berubah menjadi year-month-date hour-minute-second. Oleh karena itu ketika Anda menggunakan fungsi yang sama kepada kolom yang telat terupdate ini tentunya nilainya akan menjadi NA karena formatnya tidak sesuai.
Pertanyaannya adalah, ketika Anda telah mengubah format dari data di awal dan menimpa kolom lama dengan kolom baru, untuk apa Anda melakukan proses yang sama kepada kolom yang telah berubah menjadi format baru?
Demikian yang dapat kami sampaikan. Semoga membantu :)

**Reply by DeanAlexander27** [reply]
mantap ka Kevin, terima kasih ka sudah ter solved.
